.gitignore

```diff
@@ -1,3 +1,51 @@
+### JetBrains template
```

Comment:
Jepp. You are right. My very own .gitignore should have not benn part of the commit.